cabbage/worker.py

```diff
@@ -139,8 +139,10 @@ def run_job(self, job: jobs.Job) -> None:
 
         logger.info("Starting job", extra={"action": "start_job", "job": log_context})
         try:
-            task(**job.task_kwargs)
+            # We don't know if task return something and we don't really care
+            task_result = task(**job.task_kwargs)  # type: ignore
```

Comment:
In this case, task should return `Any` ?

Reply:
Yes, you're right :+1: 

(First time with mypy :>)

Reply:
:+1: